content/contracts-cairo/index.mdx

```diff
@@ -76,11 +73,11 @@ openzeppelin = "3.0.0-alpha.2"
 The previous example would import the entire library. We can also add each package as a separate dependency to
 improve the building time by not including modules that won’t be used:
 
-```javascript
+```toml
 [dependencies]
-openzeppelin_access = "3.0.0-alpha.2"
-openzeppelin_token = "3.0.0-alpha.2"
-openzeppelin_interfaces = "1.0.0"
+openzeppelin_access = "3.0.0-alpha.1"
+openzeppelin_token = "3.0.0-alpha.1"
+openzeppelin_interfaces = "{{openzeppelin_interfaces_version}}"
```

Comment:
The plugin allos us to do this without adding just by adding constants to the replacement.ts file, configurable per enviroment.